cloudinary-core/src/main/java/com/cloudinary/transformation/RadiusOption.java

```diff
@@ -0,0 +1,29 @@
+package com.cloudinary.transformation;
+
+import com.cloudinary.utils.StringUtils;
+
+public class RadiusOption {
+    public static String toExpression(Object[] radiusOption) {
+        if (radiusOption == null) {
+            return null;
+        }
+
+        if (radiusOption.length == 0 || radiusOption.length > 4) {
+            throw new IllegalArgumentException("Radius array should contain between 1 and 4 values");
+        }
+
+        return StringUtils.join(normalizeRadiusOptions(radiusOption), ":");
+    }
+
+    private static Object[] normalizeRadiusOptions(Object[] radiusOptions) {
```

Comment:
As most transformation params support explicit string, we (mostly) don't do a lot of validations. 
Furthermore - if there is a bad input, we can't just replace it with zeroes. 

We should only validate that there are no nulls before calling `StringUtils.join()`, and remove `normalizeRadiusOptions` method.